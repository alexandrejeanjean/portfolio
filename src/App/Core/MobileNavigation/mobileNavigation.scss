.mobile-navigation-wrapper {
  display: none;
}
@media screen and (max-width: 768px) {
  .mobile-navigation-wrapper {
    width: 95vw;
    height: 70px;
    position: fixed;
    z-index: 100;
    bottom: 10px;
    left: 0;
    right: 0;
    margin: auto;
    display: flex;
    justify-content: space-evenly;
    align-items: center;
    background-color: rgb(255, 255, 255);
    box-shadow: 0px 8px 40px rgba(0, 0, 0, 0.1);
    border-radius: 30px;
  }
}
