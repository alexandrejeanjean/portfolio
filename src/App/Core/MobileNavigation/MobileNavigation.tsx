import React from 'react'
import NavButtons from '../../Shared/NavButtons/NavButtons'
import './mobileNavigation.scss'

type TMobileNavigation = { path: string }

const MobileNavigation = ({ path }: TMobileNavigation) => {
  return (
    <>
      <nav className='mobile-navigation-wrapper'>
        <NavButtons text='Projets' position='left' path={path} isMobile />
        <NavButtons
          text='Faisons connaissance'
          position='bottom'
          path={path}
          isMobile
        />
        <NavButtons text='CompÃ©tences' position='right' path={path} isMobile />
      </nav>
    </>
  )
}

export default MobileNavigation
