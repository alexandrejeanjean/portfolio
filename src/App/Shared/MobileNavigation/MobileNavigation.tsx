import React from 'react'
import './mobileNavigation.scss'

const MobileNav = () => <header className='header'></header>

export default MobileNav
