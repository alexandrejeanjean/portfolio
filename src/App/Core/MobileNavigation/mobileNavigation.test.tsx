import React from 'react'
import { shallow } from 'enzyme'

import MobileNavigation from './MobileNavigation'

const NavigationProps = {
  path: '/projets',
}

describe('MobileNavigation component', () => {
  it('should render MobileNavigation component correctly', () => {
    const NavigationWrapper = shallow(
      <MobileNavigation path={NavigationProps.path} />
    )
    expect(NavigationWrapper).toMatchSnapshot()
  })
})
