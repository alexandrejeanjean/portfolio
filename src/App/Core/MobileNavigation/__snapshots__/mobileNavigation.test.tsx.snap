// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileNavigation component should render MobileNavigation component correctly 1`] = `
<Fragment>
  <nav
    className="mobile-navigation-wrapper"
  >
    <NavButtons
      isMobile={true}
      path="/projets"
      position="left"
      text="Projets"
    />
    <NavButtons
      isMobile={true}
      path="/projets"
      position="bottom"
      text="Faisons connaissance"
    />
    <NavButtons
      isMobile={true}
      path="/projets"
      position="right"
      text="CompÃ©tences"
    />
  </nav>
</Fragment>
`;
